### PR TITLE
migrate to dmsghttp on binaries

### LINKED ISSUE #1070: Move migration code into Debian scripts
**Feature description**

We currently handle config migrations and other upgrading procedures in the Skywire Visor binary as we previously did not have a pacakging system available to do these checks with. That is less than optimal and should be moved out of the binary and into the Debian scripts as it is easier to handle and should be less error prone hopefully.

### PR DESCRIPTION
**Pull request by mrpalide**
Did you run `make format && make check`? Yes

Fixes #1070 

 Changes:	
- Add migration to dmsghttp by checking local location in skywire-visor running

How to test this PR:
1. Open `https://ipinfo.io/country`
2. Change line 44 in `/internal/netutil/netutil.go` to result of step 1
3. Build binaries
4. Run `./skywire-cli config gen`
5. Run `./skywire-visor -c skywire-config.json`
6. Check `skywire-config.json` and see changes on addresses.

### CODE REVIEW

File: internal/netutil/netutil.go [hunk 1]
```diff
@@ -26,3 +30,19 @@ func LocalAddresses() ([]string, error) {
 
 	return result, nil
 }
+
+// LocalProtocol check a condition to use dmsghttp or direct url
+func LocalProtocol() bool {
+	resp, err := http.Get("https://ipinfo.io/country")
```
Comment (ersonp) [inline internal/netutil/netutil.go:36]: As this `ip.skycoin.com` is used everywhere for location data, I think we should use it here too.

**Comment by ersonp** [comment]
Migration is working, we need to deploy `dmsghttp` enabled services and add their PK's to `dmsghttp-config.json` as currently it has dummy values `dmsg://pk:port`.

File: internal/netutil/netutil.go (hunk 1 above)
Comment (mrpalide) [reply internal/netutil/netutil.go:36]: I didn't check our service, but I checked `ipinfo.io` by some guys in our mainnet telegram group and work well there.
We can change ours if can confirm that work fine there.

Comment (ersonp) [reply internal/netutil/netutil.go:36]: I forgot about that issue. I think it's better to just keep it `ipinfo.io`.

**Review by jdknives — APPROVED** [review]

**Pull request merged**